argo-cd-apps/base/member/infra-deployments/o11y/011y.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: o11y
+spec:
+  generators:
+    - merge:
+        mergeKeys:
+          - nameNormalized
+        generators:
+          - clusters:
+              values:
+                sourceRoot: components/o11y
+                environment: staging
+                clusterDir: base
```

Comment:
@Omeramsc it won't work like this. You need to add the list generator as well. See https://github.com/redhat-appstudio/infra-deployments/blob/main/argo-cd-apps/base/member/infra-deployments/authentication/authentication.yaml#L16

Reply:
ack, fixed. is that sufficient?